scripts/build/add-basic-constructor-to-react-component.js

```diff
@@ -0,0 +1,40 @@
+// this is to avoid the spread argument helper for react component constructors
+module.exports = function(babel) {
+  const { types: t } = babel
+
+  return {
+    name: 'ast-transform', // not required
+    visitor: {
+      Class(path) {
+        if (
```

Comment:
could be made more readable with `buildMatchMemberExpression` & I would extract `hasConstructor` too
```js
const isReactComponent = t.buildMatchMemberExpression('React.Component')

// ...

if (isReactComponent(path.node.superClass) && hasConstructor(path.node.body.body)) {
  // ...
}
```